src/table/components/TableHeadWrapper.tsx

```diff
@@ -61,7 +73,7 @@ function TableHeadWrapper({
 
           return (
             <TableCell
-              sx={headerStyle}
+              style={style}
```

Comment:
if you are dragging the column, the width updates very frequently. If we would use `sx`, mui would create a new class for every width value. That throws warnings and would blow  up the head in the DOM